en/simulation/multi-vehicle-simulation.md

```diff
@@ -39,88 +38,72 @@ To test an example setup, follow the below steps,
-                    <arg name="vehicle" value="$(arg vehicle)"/>
-                    <arg name="rcS" value="$(arg rcS1)"/>
-                    <arg name="mavlink_udp_port" value="14560"/>
-                    <arg name="ID" value="$(arg ID)"/>
-                </include>
-    ```
     
-  * runs a mavros node,
-    ```
-            <include file="$(find mavros)/launch/node.launch">
-            <arg name="pluginlists_yaml" value="$(arg pluginlists_yaml)" />
-            <arg name="config_yaml" value="$(arg config_yaml)" />
-
-            <arg name="fcu_url" value="$(arg fcu_url)" />
-            <arg name="gcs_url" value="$(arg gcs_url)" />
-            <arg name="tgt_system" value="$(arg tgt_system)" />
-            <arg name="tgt_component" value="$(arg tgt_component)" />
-        </include>
-        ```
-* The complete block for each vehicle looks like the following,
-    ```
-        <!-- UAV2 iris_2 -->
-    <group ns="uav2">
-        <arg name="fcu_url" default="udp://:14541@localhost:14559"/>
-        <arg name="gcs_url" value=""/>
-        <arg name="tgt_system" value="2"/> 
-        <arg name="tgt_component" value="1"/>
-        <arg name="rcS2" default="$(find px4)/posix-configs/SITL/init/$(arg est)/$(arg vehicle)_2"/>
-        <arg name="ID" value="2"/>
-
-        <include file="$(find px4)/launch/single_vehcile_spawn.launch">
-            <arg name="x" value="1"/>
-            <arg name="y" value="0"/>
-            <arg name="z" value="0"/>
-            <arg name="R" value="0"/>
-            <arg name="P" value="0"/>
-            <arg name="Y" value="0"/>
-            <arg name="vehicle" value="$(arg vehicle)"/>
-            <arg name="rcS" value="$(arg rcS2)"/>
-            <arg name="mavlink_udp_port" value="14562"/>
-            <arg name="ID" value="$(arg ID)"/>
-        </include>
-
-        <include file="$(find mavros)/launch/node.launch">
-            <arg name="pluginlists_yaml" value="$(arg pluginlists_yaml)" />
-            <arg name="config_yaml" value="$(arg config_yaml)" />
-
-            <arg name="fcu_url" value="$(arg fcu_url)" />
-            <arg name="gcs_url" value="$(arg gcs_url)" />
-            <arg name="tgt_system" value="$(arg tgt_system)" />
-            <arg name="tgt_component" value="$(arg tgt_component)" />
-        </include>
-    </group>
-    ```
-To simulate more vehicles, add more blocks with the appropriate changes in the startup files, and launch file's parameters.
\ No newline at end of file
+  * runs a mavros node
+  ```
+      <!-- MAVROS -->
+      <include file="$(find mavros)/launch/px4.launch">
+          <arg name="fcu_url" value="$(arg fcu_url)"/>
+          <arg name="gcs_url" value=""/>
+          <arg name="tgt_system" value="$(arg ID)"/>
+          <arg name="tgt_component" value="1"/>
+      </include>
+  ```
+  * Note: The complete block for each vehicle is enclosed in a set of `<group>` tags to seperate the ROS namespaces of the vehicles.
```

Comment:
I'll fix up this note as a post-process. Hard to get right if you're not rendering this using gitbook toolchain.